connectors/sources/google_cloud_storage.py

```diff
@@ -128,6 +113,23 @@ def get_default_configuration(cls):
             },
         }
 
+    async def validate_config(self):
+        """Validates whether user inputs are valid or not for configuration field.
+
+        Raises:
+            Exception: The service account json is empty.
+            Exception: The format of service account json is invalid.
+        """
+        if (
+            self.configuration["service_account_credentials"] == ""
+            or self.configuration["service_account_credentials"] is None
+        ):
+            raise Exception("Google Cloud service account json can't be empty.")
+        try:
+            _ = json.loads(self.configuration["service_account_credentials"])
+        except ValueError:
```

Comment:
From the [doc](https://docs.python.org/3/library/json.html#json.loads), it raises `JSONDecodeError`:

>If the data being deserialized is not a valid JSON document, a [JSONDecodeError](https://docs.python.org/3/library/json.html#json.JSONDecodeError) will be raised.

Reply:
[JSONDecodeError ](https://docs.python.org/3/library/json.html#json.JSONDecodeError)is the Subclass of [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError).

Since we are raising the custom error message & to avoid an extra import, we used ValueError to catch decode errors. 

Still Let me know if it needs to be changed to `JSONDecodeError`, I will update it.